src/components/dialogroom/card/CardRoomPlayerName.vue

```diff
@@ -19,7 +19,7 @@
                             id="inputPlayerName"
                             :value="name"
                             @input="setPlayerName"
-                            maxlength="10"
+                            maxlength="20"
```

Comment:
You should change tag size, full player name is not displayed ...
![image](https://user-images.githubusercontent.com/20130405/204103063-ebd9267b-0802-4099-ab22-7c2b258a0e8b.png)